README.md

```diff
@@ -1,278 +1,313 @@
+
 ```yml
-models:
-  <my_project>
-    materialized: 'table'
-    table_type =  ‘fact’ 
-    primary_index = ['customer_id', 'first_name']
+seeds:
+  +quote_columns: false  #or `true` if you have csv column headers with spaces
 ```
 
-|     Field     |                                                     Description                                                      |
-|---------------|----------------------------------------------------------------------------------------------------------------------|
-| `materialized`  | Use `table`                                                                                                            |
-| `table_type`    | Can be either `fact` or `dimension`                                                                                  |
-| `primary_index` | Required for fact tables and can be either a string or a list of strings for the column names of your primary index. |
+#### Supporting Concurrent Development
 
-Read more in [dbt docs on configuring models](https://docs.getdbt.com/docs/building-a-dbt-project/building-models#configuring-models)
+In dbt, database schemas are used to compartmentalize developer environments so that concurrent development does not cause table name collisions. Firebolt, however, does not currently support database schemas (it is on the roadmap). To work around this, we recommend that you add the following macro to your project. This macro will take the `schema` field of your `profiles.yml` file and use it as a table name prefix.
 
-### Join and Aggregating Indexes
+```sql
+-- macros/generate_alias_name.sql
+{% macro generate_alias_name(custom_alias_name=none, node=none) -%}
+    {%- if custom_alias_name is none -%}
+        {{ node.schema }}__{{ node.name }}
+    {%- else -%}
+        {{ node.schema }}__{{ custom_alias_name | trim }}
+    {%- endif -%}
+{%- endmacro %}
+```
 
-In addition to [primary indexes](https://docs.firebolt.io/concepts/get-instant-query-response-time#how-to-choose-your-primary-index), Firebolt also supports:
-- [Aggregating](https://docs.firebolt.io/concepts/get-instant-query-response-time#get-sub-second-query-response-time-using-aggregating-indexes)
-- [Join](https://docs.firebolt.io/concepts/get-instant-query-response-time#accelerate-joins-using-join-indexes)
+For an example of how this works, let’s say Shahar and Eric are both working on the same project.
 
-dbt-firebolt follows the same convention for indexes as was introduced to dbt-postgres ([more info on indexes usage in dbt-postgres](https://docs.getdbt.com/reference/resource-configs/postgres-configs#indexes)).
+In their `.dbt/profiles.yml`, Sharar sets `schema=sh`, whereas Eric sets `schema=er`. When they each run the `customers` model, the models will land in the database as tables named `sh_customers` and `er_customers`, respectively. When running dbt in production, you would use yet another `profiles.yml` with nothing set in the `schema` field (or any string of your choice).
```

Comment:
```suggestion
In her `.dbt/profiles.yml`, Sharar sets `schema=sh`, whereas Eric sets `schema=er` in his. When each runs the `customers` model, the models will land in the database as tables named `sh_customers` and `er_customers`, respectively. When running dbt in production, you would use yet another `profiles.yml` with nothing set in the `schema` field (or any string of your choice).
```

Reply:
I don't understand the last sentence. Why would it be different in production? And we said above that `schema` was a necessary field, but now I think you're saying you can set it as nothing.

Reply:
Perhaps a fact-check is needed here - is `schema` actually required in the profile? What would I do if don't want a prefix on prod?

Reply:
It's a required field. dbt will refuse to run it without it. It also can't be blank; I tried.